apps/hash-frontend/src/pages/shared/entity-type-page.tsx

```diff
@@ -122,7 +122,7 @@ export const EntityTypePage = ({
   const isDraft = !!draftEntityType;
 
   const handleSubmit = wrapHandleSubmit(async (data) => {
-    if (!isDirty && isDraft) {
+    if (!isDirty && !isDraft) {
```

Comment:
Driveby fix (this is what was preventing creation of draft types with no properties)